src/pydocstyle/cli.py

```diff
@@ -49,7 +49,8 @@ def run_pydocstyle():
 
     count = 0
     for error in errors:
-        sys.stdout.write('%s\n' % error)
+        if hasattr(error, 'code'):
```

Comment:
`errors` contain 2 object types - Exceptions (caught by pydocstyle) and errors (generated by pydocstyle). If some day `errors` would contain more than those 2 object types the `hasattr` check might not be good enough, but my proposal might not be good as well. Leave it as it is.